src/main/resources/changelog/db.changelog-master.xml

```diff
@@ -9,17 +9,49 @@
   http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd">
 
     <changeSet author="Mazur" id="changelog-1.0">
-
         <createTable tableName="account">
             <column autoIncrement="true" name="id" type="INTEGER">
                 <constraints nullable="false" primaryKey="true"/>
             </column>
             <column name="name" type="VARCHAR(255)"/>
             <column name="last_name" type="VARCHAR(255)"/>
             <column name="age" type="INTEGER"/>
+            <column name="country" type="INTEGER"/>
```

Comment:
don't touch the existing `changeSet`, if you need to do some changes, need to create a new chanset.
and why change from country_id to country?